scripts/waffles.coffee

```diff
@@ -37,6 +38,17 @@ module.exports = (robot) ->
           order_name != name and not order_name.endsWith " _via #{name}_"
       robot.brain.set waffleType, nameList
 
+  endOrder = (msg) ->
+    totalWaffles = (robot.brain.get(waffleType).length for waffleType in waffleTypes)
+      .reduce (x,y)->x+y
+    waitTime = totalWaffles * WAFFLE_MAKETIME
+    setTimeout (->
+      msg.send 'You can go down to collect the waffles nao nao!!~~'
```

Comment:
Can you see if using `msg.reply` here instead of `msg.send` will get the bot to tag/mention the person who originally started waffles mode? That would make sure they see this important message, while not resorting to using `@here`!

Reply:
Good point! We will want to inform the initiator. However I think we have to store the original `msg` inside `robot.brain` as `endOrder` could be triggered by "waffles stop", which will result in that person being tagged >_< I will update it accordingly.